sdk/storage/Azure.Storage.Blobs/api/Azure.Storage.Blobs.netstandard2.0.cs

```diff
@@ -51,6 +51,7 @@ public partial class BlobClientOptions : Azure.Core.ClientOptions
     {
         public BlobClientOptions(Azure.Storage.Blobs.BlobClientOptions.ServiceVersion version = Azure.Storage.Blobs.BlobClientOptions.ServiceVersion.V2020_08_04) { }
         public Azure.Storage.Blobs.Models.CustomerProvidedKey? CustomerProvidedKey { get { throw null; } set { } }
+        public bool DisableTenantDiscovery { get { throw null; } set { } }
```

Comment:
A couple of questions:
- How often will this appear?  Should it just go on our base `ClientOptions` since disabling tenant discovery is kind of a no-op for anything not supporting it?
- Should/could we group this with other auth related options so it's not a top level property?  I don't think we were strict enough about this in Storage when getting started, but I'm trying hard to clamp down on the number of top level ClientOptions properties so customers have an easier time getting to the critically important stuff.

Reply:
- My only hesitation with adding it to ClientOptions is that it does not apply to clients that don't support Token auth. Perhaps that is a small enough subset that this is not a concern.
- Perhaps this should live on `TokenCredentialOptions`? //cc @schaabs